app/components/node/PoSSize.tsx

```diff
@@ -142,6 +152,28 @@ const PoSSize = ({
     setNumUnit(commitments[index].numUnits);
   };
 
+  const validate = (value: number) => {
+    if (value < DEFAULT_POS_MAX_FILE_SIZE || Number.isNaN(value)) {
+      return DEFAULT_POS_MAX_FILE_SIZE;
+    }
+    if (value > DEFAULT_POS_MAX_FILE_SIZE * 50) {
+      return DEFAULT_POS_MAX_FILE_SIZE * 50;
+    }
+    return value;
+  };
+
+  const handleOnChange = ({ value }) => {
+    const valueInBytes = covertGBToBytes(parseInt(value, 10));
+    if (valueInBytes > DEFAULT_POS_MAX_FILE_SIZE * 50) {
+      return;
```

Comment:
It still makes it impossible to type like `999` and then round it to `100` on the blur event. So better to have
```suggestion
      setMaxFileSize(DEFAULT_POS_MAX_FILE_SIZE_LIMIT);
      return;
```
Or just change some temporary value on change and handle it only on blur :)